src/main/java/gov/usgs/owi/nldi/controllers/LinkedDataController.java

```diff
@@ -153,7 +153,13 @@ public void getCharacteristicData(HttpServletRequest request, HttpServletRespons
 			@RequestParam(value=Parameters.CHARACTERISTIC_ID, required=false) String[] characteristicIds) throws IOException {
 		BigInteger logId = logService.logRequest(request);
 		try (CharacteristicDataTransformer transformer = new CharacteristicDataTransformer(response)) {
+
 			String comid = getComid(featureSource, featureID);
+
+			if (null == comid) {
+				response.sendError(HttpStatus.NOT_FOUND.value(), "Feature not found");
+			}
+
```

Comment:
This is the key change here:  Another place where a 500 would have been thrown instead of a 404.